### PR TITLE
update registration fileError banner to notification

### PR DESCRIPTION
**Pull request by jaserud**
**Description of changes**

- changed the registration fileErrors display from banner to notification
 - made appropriate changes for this ^
- msg -> message in clinical submission (for consistency)
- updated gqls as needed

connected to PR https://github.com/icgc-argo/platform-api/pull/111
  -  tests are failing because schema updated ^
  - `npm run test-gql-validation` is working in local setup

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [ ] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [ ] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

**Comment by jaserud** [comment]
this should be merged with https://github.com/icgc-argo/platform-api/pull/111 & https://github.com/icgc-argo/argo-clinical/pull/250

File: components/pages/submission-system/program-sample-registration/index.tsx [hunk 1]
```diff
@@ -42,20 +40,33 @@ export default function ProgramIDRegistration() {
     query: { shortName: programShortName },
   } = usePageContext();
 
-  const { data: { clinicalRegistration = undefined } = {}, loading, refetch } = useQuery<{
+  const {
+    data: { clinicalRegistration = undefined } = {},
+    loading,
+    refetch,
+    updateQuery: useRegistrationSubmissionQuery,
```
[the remark this answers is not in the document]
Comment (hlminh2000) [reply components/pages/submission-system/program-sample-registration/index.tsx]: @jaserud FYI for your reference :) there's a convention about the word `"use"` in naming stuff with React https://github.com/icgc-argo/platform-api/pull/111#pullrequestreview-318553917

File: components/pages/submission-system/program-sample-registration/index.tsx [hunk 2]
```diff
@@ -42,20 +40,33 @@ export default function ProgramIDRegistration() {
...
+  } = useQuery<{
     clinicalRegistration: ClinicalRegistration;
   }>(GET_REGISTRATION, {
     variables: { shortName: programShortName },
   });
 
-  const fileErrors = get(clinicalRegistration, 'errors', [] as typeof clinicalRegistration.errors);
+  const schemaAndValidationErrors = get(
```
[the remark this answers is not in the document]
Comment (jaserud) [reply components/pages/submission-system/program-sample-registration/index.tsx]: There is validation step in registration too, it just happen after schema checks. I guess its one or  the other so `schemaOrValidationErrors`?? not sure...

[the remark this answers is not in the document]
Comment (hlminh2000) [reply components/pages/submission-system/program-sample-registration/index.tsx]: hmm I see, and those do get persisted right?

File: components/pages/submission-system/program-sample-registration/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jaserud) [reply components/pages/submission-system/program-sample-registration/index.tsx]: how about `updateRegistrationSubmisisonQuery` following how you did it with `updateClinicalSubmissionQuery`

File: components/pages/submission-system/program-sample-registration/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (jaserud) [reply components/pages/submission-system/program-sample-registration/index.tsx]: no in registraiton it is not persisted.

File: components/pages/submission-system/program-sample-registration/index.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (hlminh2000) [reply components/pages/submission-system/program-sample-registration/index.tsx]: oh lol sorry I mistyped. `RegistrationSubmisison` sounds confusing be cause both `Registration` and `Submission` refer to two different things for us... `updateClinicalRegistrationQuery` maybe?

[the remark this answers is not in the document]
Comment (jaserud) [reply components/pages/submission-system/program-sample-registration/index.tsx]: ^ lol that's what I meant to type

File: components/pages/submission-system/program-sample-registration/index.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (hlminh2000) [reply components/pages/submission-system/program-sample-registration/index.tsx]: ah ok :) sounds good to me then. Just out of curiosity, what other errors do we have in `fileErrors` other than the file name?

**Pull request merged**